test/e2e/specs/plugins/plugins__browse-calypso-user.ts

```diff
@@ -102,5 +102,23 @@ describe( DataHelper.createSuiteTitle( 'Plugins: Browse' ), function () {
 				await pluginsPage.validateHasSection( section );
 			}
 		);
+
+		it( 'Breadcrum update site when switch site', async function () {
```

Comment:
```suggestion
		it( 'The site param in each breadcrumb url updates when the user switches between sites', async function () {
```

It's good to assume whoever gets to read this failure will have no idea about the implementation so any hints we can give them about what broke goes a long way.

Reply:
Thanks @lsl !